index.html

```diff
@@ -0,0 +1,313 @@
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;400;500;700&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/1bc3a7e061.js" crossorigin="anonymous"></script>
+    <title>Mobile first Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <!-- HEADER STARTS HERE -->
+  <header class="portfolio-header">
+    <nav class="navigation">
+      <div class="portfolio-logo">
+        <a href="#" class="portfolio-text">My Portfolio</a>
+      </div>
+      <div class="menu">
+        <ul class="menu-items">
+          <li class="menu-item"><a href="#">Portfolio</a></li>
+          <li class="menu-item"><a href="#">About</a></li>
+          <li class="menu-item"><a href="#">Contact</a></li>
+        </ul>
+      </div>
+      <div class="hamburger-menu">
+        <a href="#">
+          <i class="fa-solid fa-bars"></i>
+        </a>
+      </div>
+    </nav>
+    <div class="headline">
+      <div class="headline-container">
+        <div class="headline-header">
+          <h1 class="portfolio-owner">I'm Belay <br> Glad to see you!</h1>
+          <p class="about-the-author">I’m a software developer! I can help you build a product , feature or website Look through
+            some of my work and
+            experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        </div>
+        <div class="social-media-links">
+          <h2>Let's Connect</h2>
+          <ul class="social-media-icons">
+            <li class="social-media-icon"><a href="https://twitter.com/2belamit" aria-label="Twitter" target="_blank"
+                rel="noopener"><i class="fa-brands fa-twitter"></i></a></li>
+            <li class="social-media-icon"><a href="https://www.linkedin.com/in/belay-birhanu-144ba714b/" aria-label="Linkedin"
+                target="_blank" rel="noopener"><i class="fa-brands fa-linkedin-in"></i></a></li>
+            <li class="social-media-icon"><a href="https://github.com/belaymit?tab=repositories" aria-label="Github"
+                target="_blank" rel="noopener"><i class="fa fa-github"></i></a></li>
+            <li class="social-media-icon"><a href="#" aria-label="Angellist" target="_blank" rel="noopener"><i
+                  class="fa-brands fa-angellist"></i></a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </div>
+  </header>
+<!-- END OF HEADER -->
+<!-- Work Section starts here -->
+  <section class="work-section">
+    <div class="projects-card">
+      <div class="card-header">
+          <div class="projects">
+            <img src="./Images/Snapshoot Portfolio__.png" alt="" class="project-screenshot">
+          </div>
+      </div>
+      <div class="card-body">
+        <div class="author-role-company">
+          <h1 class="project-title">Tonic</h1>
```

Comment:
**OPTIONAL** - To improve the SEO of your website, it is not advisable to have more than one `h1` tag on an HTML page, please ensure that you have at least one, but no more than one `h1` tag on your page 👍 👍 

**OPTIONAL** - When using `hx` tags, it is essential to follow hierarchy i.e use an `h2` tag after an `h1` tag, or use an `h3` tag after an `h2` tag. This would help improve your site's accessibility. Please fix this for your page 😄 😄